geograypher/meshes/meshes.py

```diff
@@ -852,6 +862,25 @@ def get_values_for_verts_from_vector(
             return labeled_verts, all_values
         # Else return a dict of all requested values
         return labeled_verts_dict, all_values_dict
+    
+    def save_IDs_to_labels(self, savepath: PATH_TYPE):
+        """saves the contents of the IDs_to_labels to the file savepath provided
+
+        Args:
+            savepath (PATH_TYPE): path to the file where the data must be saved
+        """
+
+        # Save the classes filename
+        savepath = Path(savepath)
+        savepath.parent.mkdir(parents=True, exist_ok=True)
```

Comment:
Can you use the `ensure_containing_folder` function?

Reply:
Done.